backend/remote-state/swift/client.go

```diff
@@ -13,23 +13,19 @@ import (
 	"github.com/hashicorp/terraform/state/remote"
 )
 
-const (
-	TFSTATE_NAME      = "tfstate.tf"
-	TFSTATE_LOCK_NAME = "tfstate.lock"
-)
-
 // RemoteClient implements the Client interface for an Openstack Swift server.
 type RemoteClient struct {
 	client           *gophercloud.ServiceClient
 	container        string
+	state_name       string
```

Comment:
please change this to `stateName`: https://golang.org/doc/effective_go.html#mixed-caps